package.json

```diff
@@ -37,6 +37,7 @@
     "babel-plugin-transform-es3-property-literals": "6.8.0",
     "babel-plugin-transform-flow-strip-types": "^6.18.0",
     "babel-plugin-transform-inline-environment-variables": "6.8.0",
+    "babel-plugin-transform-runtime": "^6.23.0",
```

Comment:
we have removed it some time ago, in cases like other libraries using this lib as a dependency it increases the build too much.